js/main.js

```diff
@@ -12,44 +12,74 @@ const $form = document.querySelector('form');
 $form.addEventListener('submit', handleSubmit);
 
 function handleSubmit(event) {
-  event.preventDefault();
   const formInfo = {
     title: $form.elements.title.value,
     photoURL: $form.elements.photoURL.value,
     notes: $form.elements.notes.value,
     entryId: data.nextEntryId,
   };
-  data.entries.unshift(formInfo);
-  $ul.prepend(renderEntry(formInfo));
+  if (data.editing === null) {
+    event.preventDefault();
+    data.entries.unshift(formInfo);
+    $ul.prepend(renderEntry(formInfo));
+    $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+    data.nextEntryId++;
+    toggleNoEntries();
+    $form.reset();
```

Comment:
This should get called outside any conditionals, so that way it gets reset after we're done editing and after we created a new entry